### PR TITLE
Add practice with gans

### PR DESCRIPTION
**Pull request by sovrasov**
There is no model for now, will be added soon.

### CODE REVIEW

**Comment by dkurt** [comment]
I think it's better to rename it to `9_gan` (`9_colorization` is also related to GAN but demonstrates a bit different approach)

File: modules/9_gan/test_generator.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+all_images = np.zeros((img_size * grid_size, img_size * grid_size, 3), dtype=np.uint8)
+
+for i in range(grid_size * grid_size):
+    generated = model.generate()
+    row = i // grid_size
+    col = i % grid_size
+    all_images[img_size * row : img_size * (row + 1),
+               img_size * col : img_size * (col + 1), :] = generated
+
+cv.imwrite('../../data/generated_img.png', all_images)
```
Comment (dkurt) [inline modules/9_gan/test_generator.py]: Please use environment variable to determine data folder: 
```python
cv.imwrite(os.path.join(data_folder, 'generated_img.png'), all_images)
```

**Comment by dkurt** [comment]
Feel free to share a model with us so we will publish on Mega

**Comment by sovrasov** [comment]
[Model](https://drive.google.com/drive/folders/18lDnBUtd_mgQHn_ckaUwERwU2IS9I4gy?usp=sharing)

File: modules/9_gan/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Original images from CelebA (left), generated images (right).
+    <img src="../../data/celeba_samples.jpeg" width="320">
+    <img src="../../data/generated_img.png" width="320">
+
+## Details
+
+* Download converted model from here:
+
+  * [celeba_generator.xml](https://mega.nz/#!hAslCCaL!2vWXjiPq3EbuLZbOvEDOI0y9VorGn5enUm7UMNWGrWo)
+  * [celeba_generator.bin](https://mega.nz/#!AR1HwIIA!U3p46tM9SFMhKzbSwJIEiugdJDQ5mUSOC3twZL6UYTM)
```
Comment (dkurt) [inline modules/9_gan/README.md]: Please add the links to `.travis.yml`

File: modules/9_gan/test_generator.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
```
Comment (dkurt) [inline modules/9_gan/test_generator.py]: Please rename this file to `test_gan`. Testing infrastructure tries to run a script with the same name as module.

Make sure that `chmod +x test_gan.py` is applied.

**Pull request merged**